### DISCUSSION TITLE
不具合報告: Markdownで、URLの単独投稿が水平線に変換されてしまう

### DISCUSSION THREAD

**Discussion by ffunatsu** [increments/qiita-discussions#929] [要望・アイデア・不具合]
もし重複があればすみません。

Markdownで、以前は `[xxx](yyy)` 形式でない単独のURL投稿（例えば単に `http://example.com/test` など）が許可されていたのですが、現在単独でURLを書くと、なぜか水平線に変換され、URL自体は見えなくなり、私の書いていた記事の多くが一見何のことを示しているのかよくわからないものになってしまいました。

これはもしかすると、いわゆるプレビューウィジェットの生成に失敗している（サムネや概要が取得できないような）URLがこうなってしまうのかもしれませんが、内部で何が起こっているのかは私にはわかりません。

#921 の修正はもしかすると関連があるかもしれません。

**Comment by ffunatsu** [comment]
こちら、投稿してから気付いたのですが、もしかすると個人的に使っているChrome拡張機能などとの兼ね合いがあるのかもしれません。ただ、記事の多くは既に修正してしまったので、検証環境が残っておらず…。

**Comment by ohakutsu** [comment]
@funatsufumiya 
不具合報告ありがとうございます。

こちら再現できないかを試してみたのですが、再現できませんでした :bow:
もし、また同じような問題が起こるようでしたら、またご報告いただけると幸いです！

**Reply by ffunatsu** [reply]
ご確認ありがとうございます。やはりそうですよね…。どの拡張機能との相性かがちょっと不明なままではありますが、こちらクローズしていただいて問題ないのではないかと思います。

**Reply by ohakutsu** [reply]
@funatsufumiya 
承知しました！
こちらのDiscussionはクローズさせていただきます :pray:
